### PR TITLE
Working branch

### PR DESCRIPTION
**Pull request by vflorin1995**
## Add files from trials

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    	<title>Tribute Page</title>
+		<link rel="stylesheet" href="./style.css">
+	</head>
+
+	<body>
+    	<main id="main">
+			<nav id="nav-bar">
+				<h1 id="title"> Dr. Norman Borlaug </h1>
+				<h5> The man who saved a billion lives</h5>
+			</nav>
```
Comment (vflorin1995) [inline index.html:14]: I think nav tag should not be used here in the header, you can try and replaced.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+			<div id="img-div">
+				<img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" id="image">
```
Comment (vflorin1995) [inline index.html:16]: I think is better fot Img elements to have an alt attribute.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+#tribute-info{
+    position: relative;
+    left: 400px;
+   
+}
+#link{
+    position: relative;
+    left: 300px;
+}
+
```
Comment (vflorin1995) [inline style.css:40]: Please fix linter errors

**Review by vflorin1995 — COMMENTED** [review]
Please add Linters and fix all the errors and update the readme file.

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @vflorin1995   , :wave:
### I'm Levy, your PR reviewer

Excellent :sunglasses: You did an amazing job on your first review. :+1: 
However, there are some issues that you still need to work on to go to the next project but you are almost there!


- Polite comments :heavy_check_mark: 
- Good usage inline comments :heavy_check_mark: 
## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._
- [ ] Kindly make sure to remove all typo errors in order to make your review professional.
- [ ] Kindly remember to always include a full explanation of "why", for each requested change. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

 - [OPTIONAL] Please consider using a couple of emojis and/or gifs in the review. They make the code review more human-friendly 👽🤖.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by codecaiine** [comment]
Hi @vflorin1995 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Meaningful commits :heavy_check_mark: 
- Good use of inline comments :heavy_check_mark: 

## Optional suggestions

 - [ ] **Please note that you could have marked the PR as invalid for the reason that Linter result is not green, and didn't pass checks in the PR.** :+1: 
 - [ ] I would like to recommend you use a couple of emojis and/or gifs in the review to make the code review more human-frien :smile: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (codecaiine) [reply index.html:16]: Good application of changes request :+1:

File: index.html (hunk 1 above)
Comment (codecaiine) [reply index.html:14]: Good application of changes request :+1:

**Pull request merged**